### PR TITLE
Bg fix issue with e2e

### PR DESCRIPTION
**Pull request by acedeywin**
## Description

- This PR looks into handling a situation where the helpline code contains a number. It would match all digits (0-9) and replace them with no space since Lex doesn't accept numbers.

### Checklist
- [ ] Corresponding issue has been opened
- [ ] New tests added

### Related Issues
Fixes #....

### CODE REVIEW

File: functions/channelCapture/channelCaptureHandlers.private.ts [hunk 1]
```diff
@@ -360,7 +360,13 @@ export const handleChannelCapture = async (
   ]);
 
   const { ENVIRONMENT, HELPLINE_CODE } = context;
-  const languageSanitized = language.replace('-', '_'); // Lex doesn't accept '-'
+  let languageSanitized = language.replace('-', '_'); // Lex doesn't accept '-'
+
+  // This is used to match all digits (0-9) and replace them with no space since Lex doesn't accept numbers
+  if (/\d/.test(languageSanitized)) {
```
Comment (robert-bo-davis) [inline functions/channelCapture/channelCaptureHandlers.private.ts:366]: Do we need an if wrapper? Will replace return the same string if the regex isn't found?

**Comment by robert-bo-davis** [comment]
Were we also going to go ahead and revert the github action changes that tried to set the HELPLINE_CODE env var to `ee`?

**Review by stephenhand — APPROVED** [review]
Lgtm except for @robert-bo-davis comment

File: functions/channelCapture/channelCaptureHandlers.private.ts (hunk 1 above)
Comment (acedeywin) [reply functions/channelCapture/channelCaptureHandlers.private.ts:366]: @robert-bo-davis  - If there is no number the if block is ignored. So the initial `languageSanitized` will return its original value.

**Pull request merged**